scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/TestApplyWhen.java

```diff
@@ -293,9 +300,9 @@ public void testRecurrenceWindowsCutoffMidInterval() {
       logger.debug(a.startOffset().toString());
     }
 
-    assertFalse(TestUtility.activityStartingAtTime(plan,Duration.of(1, Duration.SECONDS), activityType)); //cutting off mid interval should fail, i.e. no scheduling
+    assertTrue(TestUtility.activityStartingAtTime(plan,Duration.of(1, Duration.SECONDS), activityType)); //cutting off mid interval should fail, i.e. no scheduling
```

Comment:
Same question as above. If this is intentional, maybe we should change the associated comment.

Reply:
See answer above

Reply:
Sounds good! Maybe we can update the comment with a brief explanation

Reply:
In this example I have removed the comment: //cutting of mid interval ... The outcome of the test was already properly defined at the beginning of the method. This test is also more clear as there is more room to place the goal. Regarding TestRecurrenceBabyWindow, I have added my answer to the beginning of the method.